man/systemd.exec.xml

```diff
@@ -459,6 +459,35 @@
 
         <xi:include href="system-only.xml" xpointer="singular"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>ExtensionDirectories=</varname></term>
+
+        <listitem><para>This setting is similar to <varname>BindReadOnlyPaths=</varname> in that it mounts a file
+        system hierarchy from a directory, but instead of providing a destination path, an overlay will be set
+        up. This option expects a whitespace separated list of source directories.</para>
+
+        <para>A read-only OverlayFS will be set up on top of <filename>/usr/</filename> and
+        <filename>/opt/</filename> hierarchies. The order in which the directories are listed will determine
+        the order in which the overlay is laid down: directories specified first to last will result in overlayfs
+        layers bottom to top.</para>
+
+        <para>Each directory may be prefixed with <literal>-</literal>, in which case it will be ignored
+        when its source path does not exist. If the source path contains a <literal>:</literal>, it needs to be
+        escaped as <literal>\:</literal>. Any mounts created with this option are specific to the unit, and are
```

Comment:
It looks the colon is automatically escaped.

Reply:
yes, copied the paragraph from above - removed